client/my-sites/checkout/gsuite-nudge/index.jsx

```diff
@@ -36,16 +36,14 @@ export class GSuiteNudge extends React.Component {
 		domain: PropTypes.string.isRequired,
 		receiptId: PropTypes.number.isRequired,
 		selectedSiteId: PropTypes.number.isRequired,
+		isOnboardingFlow: PropTypes.bool.isRequired,
 	};
 
 	handleSkipClick = () => {
-		const { siteSlug, receiptId, isEligibleForChecklist } = this.props;
-
-		const destination = abtest( 'improvedOnboarding' ) === 'onboarding' ? 'view' : 'checklist';
-
+		const { siteSlug, receiptId, isEligibleForChecklist, isOnboardingFlow } = this.props;
 		page(
 			isEligibleForChecklist
-				? `/${ destination }/${ siteSlug }`
+				? `/${ isOnboardingFlow ? 'view' : 'checklist' }/${ siteSlug }`
```

Comment:
This looks like a behaviour change. Previously if the flow was `ecommerce-onboarding` or `business` then the user would end up on `/view/...`, but now they will be taken to `/checklist/...`

I think it should just be this now:
```suggestion
				? `/view/${ siteSlug }`
```

We should probably change the name of `isEligibleForChecklist` too since they'll never end up on the checklist now.

Unless we want the behaviour to change? (tbh I haven't figured out how to go down this code path in testing).

Reply:
Thanks for looking closely at this @p-jackson !!!

> The distinction between "being in the onboarding AB test cohort" and "being on the onboarding flow" can be a bit subtle.

True. I interpreted, perhaps falsely, to mean users on the onboarding flow. It's in the spirit of the original changes however: https://github.com/Automattic/wp-calypso/pull/30407

> This looks like a behaviour change. Previously if the flow was ecommerce-onboarding or business then the user would end up on /view/...

I assumed we still want `onboarding` to end up on the checklist after the **GSuite** nudge?? It's the only screen we're targeting here. (Signup > purchase domain and plan > Want Gsuite ? > go to view (if onboarding flow )

I tested and it seems to work for this condition. Unless we scrap the GSuite nudge logic altogether?  @michaeldcain do you remember if there were any specific requirements for the **GSuite** nudge aside from what I see in  https://github.com/Automattic/wp-calypso/pull/30407?

I'm inclined to park any changes for another PR.

> Unless we want the behaviour to change? (tbh I haven't figured out how to go down this code path in testing).

If you mean the signup flow destination in general, I see that that whole signup destination code has been refactored.

Seems like [checklist is the destination](https://github.com/Automattic/wp-calypso/blame/master/client/signup/config/flows.js#L53) for all flows now.

Reply:
@ramonjd there are no special requirements for GSuite nudge, I think it was just best guess of the destination at a time. We can totally do checklist for all flows now:

```suggestion
				? `/checklist/${ siteSlug }`
```

In fact I went ahead and updated the PR to reflect that.